fix for Fatal error: exception Invalid_argument("gettimeofday not implemented")
author: Kenneth Hoste (HPC-UGent)
--- configure.orig	2015-10-06 10:19:12.763826416 +0200
+++ configure	2015-10-06 10:19:56.005026703 +0200
@@ -1015,7 +1015,7 @@
 fi
 
 has_select=no
-if sh ./hasgot select && \
+if cc="$cc -wd165" sh ./hasgot select && \
    sh ./hasgot -i sys/types.h $select_include -t fd_set ; then
   echo "select() found."
   echo "#define HAS_SELECT" >> s.h
@@ -1084,7 +1084,7 @@
 fi
 
 has_gettimeofday=no
-if sh ./hasgot gettimeofday; then
+if cc="$cc -wd165" sh ./hasgot gettimeofday; then
   echo "gettimeofday() found."
   echo "#define HAS_GETTIMEOFDAY" >> s.h
   has_gettimeofday="yes"
